tests/lambda_endpoints/test_spice_indexer_lambda.py

```diff
@@ -0,0 +1,141 @@
+"""Tests for the SPICE indexer lambda."""
+
+import os
+from datetime import datetime
+
+from sds_data_manager.lambda_code.SDSCode.database import models
+from sds_data_manager.lambda_code.SDSCode.pipeline_lambdas import spice_indexer
+
+
+def put_local_file_in_bucket(s3_client, path_in_s3, path_local):
```

Comment:
Are we able to use the imap-data-access upload/download routines instead of this explicit put?

Reply:
Would those work for mocked buckets? This is just to upload the local testing files into the mocked s3 bucket for unit testing

Reply:
Ahh, I didn't look closely enough that this was for the testing. Good call!